### ISSUE TITLE
Please add option for default sidebar panel.

### ISSUE THREAD

**Issue by sharvath** [Quicksaver/OmniSidebar#74]
At present, the sidebar remembers the last open panel. I would prefer for all new windows to display the bookmarks panel instead.

**Comment by Quicksaver** [comment]
Sounds reasonable. I'll see what I can do about this. But you can also just drag the bookmarks sidebar button, or use its own shortcut Ctrl+B to open it.

**Comment by sharvath** [comment]
Yes, but... well, I won't skirt the issue. I'm lazy.

These free-flowing web layouts are great for cross-platform compatibility, but they're hard to read in 16:9 (or a rough approximate in windowed mode) on a large monitor. I'd rather sacrifice part of my viewport for an ever-present bookmarks panel. That said, I can't quit digging around in options menus. I keep opening new windows that jump straight to the add-ons panel.

It's a minor nuisance, but I can't imagine I'm the only person who would appreciate the feature. Either way, I appreciate the consideration.

**Comment by Quicksaver** [comment]
You could also enable the twin sidebar and keep the bookmarks open in it (as in don't open anything else in it), that way you'll have a sidebar just for the bookmarks and another for the rest.

I don't see why it would jump straight to the add-ons panel when you open new windows. This should only happen if the last sidebar you opened actually was the add-ons panel. Is that the case?

**Comment by sharvath** [comment]
Yes, the last panel was add-ons. Sorry, I took a bit of a leap in that description.
